changelog-master.xml

```diff
@@ -5,4 +5,5 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 <include  file="sql/views.sql" relativeToChangelogFile="true"/>
+<include  file="sql/tables.sql" relativeToChangelogFile="true"/>
```

Comment:
Entries in the change-log run in a declared order. If the views depend on the tables created after this is going to fail. Additionally, there is a practice to name them with sequenced prefix 001_ 002_ etc.

Reply:
Ah yes.  I think this is horrid.  The views.sql is hardcoded to a specific schema and has already been deployed to some environments.  MySQL doesn't seem to support the forcible creation of views, so I'm not sure of the best way forward.

I could change the views.sql to be blank now - and then have 002_tables.sql and 003_views.sql?  then on a fresh deployment, the existing view.sql will be blank and so work. for an existing deployment, it would recreate the view (which I think is fine)

Is it safe to take the hardcoded schema out of the view definition? I tried it locally and it seemed fine...